llvm/lib/CheerpUtils/Utility.cpp

```diff
@@ -488,7 +493,11 @@ bool InlineableCache::isInlineableImpl(const Instruction& I)
 			case Instruction::FPToUI:
 			case Instruction::PtrToInt:
 			case Instruction::IntToPtr:
+			case Instruction::ExtractElement:
```

Comment:
I would say those 3 should all return true, and would consider adding also the specific cheerp-wasm-simd intrinsics in the list of the calls that are inlineable (If they have a single use)

Reply:
Done